.github/workflows/build-wheel-linux-x86_64.yaml

```diff
@@ -333,3 +336,5 @@ jobs:
     - name: Run Python Pytest Tests
       run: |
         python${{ matrix.python_version }} -m pytest $GITHUB_WORKSPACE/frontend/test/pytest -n auto
+        python${{ matrix.python_version }} -m pytest $GITHUB_WORKSPACE/frontend/test/pytest --backend="lightning.kokkos" -n auto
+        python${{ matrix.python_version }} -m pytest $GITHUB_WORKSPACE/frontend/test/pytest --runbraket=ON -k "not remotetests" -n auto
```

Comment:
I guess the remote tests will trigger a failure here for this test (I had to uncomment them locally), until we get your changes in @maliasadi. Alternatively we could also add the credential setup here like we do in the check-catalyst action.

Reply:
Yep, this is tackled in my PR